lib/hooks/api/useRepositoryContributors.ts

```diff
@@ -1,12 +1,12 @@
 import useSWR, { Fetcher } from "swr";
 import { publicApiFetcher } from "lib/utils/public-api-fetcher";
+import { OrderDirection } from "lib/utils/sorting";
 
 interface PaginatedResponse {
   readonly data: DbRepoContributor[];
   readonly meta: Meta;
 }
 
-type OrderDirection = "ASC" | "DESC";
```

Comment:
`type OrderDirection = "ASC" | "DESC";` was littered all over the codebase, so I consolidated it's usage everywhere into an exported type.